File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

// main
int main()
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while
    (n < 1 || n > 8);

    for (int i = 0; i < n; i++)
    {
        // Loop to add spaces  mario
        for (int k = n - i; k > 1; k--)
        {
            printf(" ");
        }
        // loop to add  the  mario #
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

    return 0;
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    n = get_int("height: ")
    if n > 0 and n < 9:
        break

# loop to handle number of rows
for i in range(n):
    # k no of spaces
    k = 1
    # loop to handle spaces
    for k in range(n - i - 1):
        print(" ", end="")
    # loop to handle number of columns
    for j in range(i+1):
        # printing #
        print("#", end="")
    # print 2 spaces
    print("  ", end="")
    # print right mario #
    for y in range(i+1):
        print("#", end="")

    print()
